test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -2071,7 +2071,7 @@ protected boolean addMockGeoShapeFieldMapper() {
      */
     protected boolean addMockTelemetryPlugin() {
         // setting to false until https://github.com/opensearch-project/OpenSearch/issues/12615 is resolved
-        return false;
+        return true;
```

Comment:
Do we need/want this to run every time? Can we randomly enable [here](https://github.com/opensearch-project/OpenSearch/blob/52ae79d017a8e961cc1bc81fa7eafcc59689a447/test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java#L2121) similar to other mock plugins?

Reply:
@Gaganjuneja @reta What do you guys think we should do there?

Reply:
We used to run it every time since this is the only "safety net" we have for it at the moment